### PR TITLE
charts(openebs-3.4.1): helm chart for OpenEBS v3.4.1

### PR DESCRIPTION
**Pull request by niladrih**
This add release v3.4.1 for the OpenEBS chart. changes:
- mayastor dependency updated to 2.0.1
- values.yaml typo fix for dynamic-nfs-provisioner options
- comment mayastor configuration set to enable the use of subchart options, by default
- remove mayastor base.initContainer and containers configuration, the subchart options will be used by default

### CODE REVIEW

**Comment by pchandra19** [comment]
Can you also please update the main README.md for this repo. 'Mayastor' does not show up as an embedded link. 

Engine charts included as dependencies are:
- [mayastor] (https://github.com/openebs/mayastor-extensions/tree/release/2.0/chart)

File: charts/openebs/values.yaml [hunk 1]
```diff
@@ -401,152 +401,142 @@ mayastor:
...
+  # loki-stack:
+  #   # -- Enable loki log collection for Mayastor components
+  #   enabled: true
+
+  # obs:
+  #   callhome:
+  #     # -- Enable callhome
+  #     enabled: true
+  #     # -- Log level for callhome
+  #     logLevel: "info"
```
Comment (avishnu) [inline charts/openebs/values.yaml:539]: Why is everything commented out?

**Review by pchandra19 — APPROVED** [review]
LGTM

**Review by avishnu — APPROVED** [review]
lgtm

**Review by tiagolobocastro — APPROVED** [review]

**Pull request merged**